### PR TITLE
refactor: changed some questions and some zod's names

### PR DESCRIPTION
**Pull request by LeticiaMalato**
##Overview
Was changed the suggestions question to match up the schema words from backend.

#How was this ?
WhatsApp Image 2023-05-21 at 17 23 45
WhatsApp Image 2023-05-21 at 17 23 46

### CODE REVIEW

File: src/hooks/useEstablishments.tsx [hunk 1]
```diff
@@ -252,3 +269,22 @@ export function useEstablishments() {
 
   return context;
 }
+
+export function createSuggestion(formData: {
```
Comment (ryanolivrdev) [inline src/hooks/useEstablishments.tsx:273]: aplicar aqui tambem a interface

File: src/hooks/useEstablishments.tsx [hunk 2]
```diff
@@ -34,6 +34,23 @@ interface EstablishmentsContextData {
   ) => Promise<void>;
   getReviews: (establishment_id: string) => Promise<IReviewEntity[]>;
   isLoading: boolean;
+
+  createSuggestion: (
```
Comment (ryanolivrdev) [inline src/hooks/useEstablishments.tsx:38]: aplicar aqui tambem a interface

**Review by ryanolivrdev — CHANGES_REQUESTED** [review]

File: src/hooks/useEstablishments.tsx [hunk 3]
```diff
@@ -252,3 +256,41 @@ export function useEstablishments() {
...
+      formData.price,
+      formData.unevenness,
+      formData.ground_floor_room,
+      formData.elevator,
+      formData.braille,
+      formData.incompatible_dimensions,
+      formData.sign_language,
+      formData.tactile_floor,
+      formData.bar
+
```
Comment (olucasfreitas) [inline src/hooks/useEstablishments.tsx:292]: remover espaço

File: src/hooks/useEstablishments.tsx [hunk 4]
```diff
@@ -14,6 +14,8 @@ import { useAuth } from './useAuth';
 import { getDistanceBetweenMotelAndUser } from '@helpers/getDistanceBetweenMotelAndUser';
 import { IEstablishmentDTO } from '@entities/establishment/establishment.dto';
 import { IReviewEntity } from '@entities/review/review.interface';
+import { ISuggestionEntity } from '@entities/suggestion/suggestion.interface';
```
Comment (olucasfreitas) [inline src/hooks/useEstablishments.tsx:17]: Isso aqui não precisa existir

File: src/screens/App/Suggestion/index.tsx [hunk 5]
```diff
@@ -32,239 +33,265 @@ export function Suggestion() {
...
+  async function handleSubmitSuggestion(formData: SuggestionData) {
+    try {
+      await createSuggestion(formData);
+      toast.show({
+        title: 'Suas informações foram enviadas com sucesso!',
+        placement: 'top'
+      });
+    } catch (error: any) {
+      toast.show({
+        title: 'Erro ao enviar as informações',
```
Comment (olucasfreitas) [inline src/screens/App/Suggestion/index.tsx:45]: te falei pra colocar o error.message aqui

File: src/hooks/useEstablishments.tsx (hunk 3 above)
Comment (LeticiaMalato) [reply src/hooks/useEstablishments.tsx:278]: 47389a57c2d9323805a6a3e112dbe5ff614981e4

File: src/hooks/useEstablishments.tsx (hunk 4 above)
Comment (LeticiaMalato) [reply src/hooks/useEstablishments.tsx]: 051ec6eae47bd1975bbe4eb34b2139a9ac316207

File: src/screens/App/Suggestion/index.tsx (hunk 5 above)
Comment (LeticiaMalato) [reply src/screens/App/Suggestion/index.tsx:46]: dd62d9d4fb4fc169c42d962eb25139683d821ff9

File: src/screens/App/Suggestion/index.tsx [hunk 6]
```diff
@@ -32,239 +33,265 @@ export function Suggestion() {
...
+          width="100%"
+          height="50px"
+          text="Enviar sugestão"
+          mb="0"
+          onPress={handleSubmit(handleSubmitSuggestion)}
         />
+      </SafeAreaWrapper>
+    );
+  }
+  function createSuggestion(formData: {
```
[the remark this answers is not in the document]
Comment (LeticiaMalato) [reply src/screens/App/Suggestion/index.tsx]: e99ffd637ade60238e650aac717db6c438c83b2d

**Review by olucasfreitas — APPROVED** [review]

**Pull request merged**